test_src/sljitTest.c

```diff
@@ -735,44 +758,54 @@ static void test9(void)
 	buf[12] = 0;
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS1(VOID, P), 4, 2, 0, 0, 0);
+	/*buf[0] */
```

Comment:
Somehow from this point there is no space between the `*` and buf. Several similar cases below.